docs/production/authentication-methods.md

```diff
@@ -289,9 +289,25 @@ AUTH_LDAP_USER_SEARCH = LDAPSearchUnion(
 
 You can restrict access to your Zulip server to a set of LDAP groups
 using the `AUTH_LDAP_REQUIRE_GROUP` and `AUTH_LDAP_DENY_GROUP`
-settings in `/etc/zulip/settings.py`. See the
-[upstream django-auth-ldap documentation][upstream-ldap-groups] for
-details.
+settings in `/etc/zulip/settings.py`.
+
+An example config for Active Directory group limiting:
+
+```
+AUTH_LDAP_REQUIRE_GROUP = "CN=ChatEnabled,CN=Users,DC=domain,DC=lan"
+AUTH_LDAP_GROUP_TYPE = ActiveDirectoryGroupType()
+AUTH_LDAP_GROUP_SEARCH = LDAPSearch("CN=Users,DC=domain,DC=lan", ldap.SCOPE_SUBTREE, "(objectClass=groupOfNames)")
+```
+
+The LDAP include must also be integrated with the group type you need for filtering. In the example above
+`ActiveDirectoryGroupType` must be added:
+
+```
+import ldap
+from django_auth_ldap.config import GroupOfNamesType, LDAPGroupQuery, LDAPSearch, ActiveDirectoryGroupType
```

Comment:
@timabbott Do you think it would make sense to try to put all possible useful imports like this in `prod_settings_template.py` rather than documenting what admins are supposed to add? (Seems like a nicer experience + I'm wondering about whether the latter option won't mess with `compare-settings-to-template` making it harder to use correctly?)

Reply:
I ended up doing `import django_auth_ldap` in the sample documentation, for readability reasons.